Lesson_4_Introduction_to_Pandas_II/Lesson_4.ipynb

```diff
@@ -6,13 +6,17 @@
    "source": [
```

Comment:
<p>Leave the following in the cell,</p><p> </p><p>titanic_data =&nbsp;</p><p>titanic_data</p><p> </p><p>This way we start the students off</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/theLeague-AI/TheLeagueAI_main/pull/23/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
<p>done</p>